utilities/payload-utils.js

```diff
@@ -99,6 +100,31 @@ PayloadUtils.prototype = {
     request(url).pipe(wstream);
   },
 
+  // Convert filestream in msg.payload to a buffer
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4764899)